pyiode/iode_database/variables_database.pyx

```diff
@@ -491,6 +493,7 @@ cdef class Variables(_AbstractDatabase):
         
         df = DataFrame(index=vars_list, columns=periods_list, data=data)
         # replace IODE NaN values by numpy or pandas values
+        # GDM> I suspect this is buggy as you do not seem to store the result, nor use inplace=True
```

Comment:
Well, the conversion IODE NaN -> Numpy NaN is already made in the function `_iodevar_to_ndarray()`.

See line above:
```python
data[i] = _iodevar_to_ndarray(_cstr(name), False)
```

I simply have to remove the df.where(...)

Reply:
See PR #557